scaling_performance/optimizing_storage.adoc

```diff
@@ -101,6 +101,15 @@ The following table summarizes the recommended and configurable storage technolo
 | Not configurable footnoteref:[object,Object storage is not consumed through {product-title}'s PVs/persistent volume claims (PVCs). Apps must integrate with the object storage REST API. ]
 |===
 
+[IMPORTANT]
+====
+For metrics it is an anti-pattern to use any shared storage and a single volume
+(RWX). By default, metrics deploys with one volume per Cassandra Replica.  
+
+For logging, one volume per loggin-es is required. Therefore using any shared
+storage would be an anti-pattern.
+====
+
```

Comment:
Can we connect this to the tabel in some way (foot note)? 

I wonder if some astrics for the file row is ndded for rwx and logging and metrics that points to this (call-out).

Reply:
Yeah, I added the statements as footnotes to logging and metrics. For the asterisks for RWX that point to the logging and metrics footnotes, I'm going to need to give it a bit more thought on how to execute that.

Thanks!